### PR TITLE
Add footer with SVGs as components

### LINKED ISSUE #34: As a user, I want to see a footer, so I have a consistent design and control over the app
### Criteria

- [ ] 
- [ ]
- [ ]

### To-Dos

- [ ] 
- [ ]
- [ ]

### PR DESCRIPTION
**Pull request by aruppert**
Closes #34

### CODE REVIEW

File: client/src/components/Footer.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import Dice from '../icons/Dice';
+import AddQuestion from '../icons/AddQuestion';
+
+const FooterBar = styled.div`
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  width: 100vw;
```
Comment (manuelwecker) [inline client/src/components/Footer.js]: Do you need `vw` if your container works with `vh`/`vw`?

File: client/src/components/Footer.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  height: 70px;
+  background: linear-gradient(
+    to right,
+    ${props => props.theme.colors.card2},
+    ${props => props.theme.colors.card1}
+  );
+  border-radius: 25px 0px 0px;
+  border-top: 3px solid #fff;
+  border-left: 3px solid #fff;
+  box-shadow: 0px 3px 6px rgba(0, 0, 0, 0.16);
```
Comment (manuelwecker) [inline client/src/components/Footer.js]: alpha 0.15? Just a suggestion

File: client/src/themes/primary.js [hunk 3]
```diff
@@ -4,7 +4,9 @@ const primary = {
     text2: '#000',
     background: '#F2F2F2',
     card1: '#04DBC6',
-    card2: '#007E71'
+    card2: '#007E71',
```
Comment (manuelwecker) [inline client/src/themes/primary.js]: think about you color names, maybe?

File: client/src/components/Footer.js (hunk 2 above)
Comment (aruppert) [reply client/src/components/Footer.js]: :D :D :D

File: client/src/themes/primary.js (hunk 3 above)
Comment (aruppert) [reply client/src/themes/primary.js]: Yeah, but 1 is shorter than primary ;)

File: client/src/components/Footer.js (hunk 1 above)
Comment (aruppert) [reply client/src/components/Footer.js]: probably would 100% work too...

**Pull request merged**